client/src/pages/Notifications/NotificationsList/NotificationsList.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Notification } from '../../../interface/Notifications';
+import timeSince from '../RelativeTimes';
+import useStyles from './useStyles';
+import Grid from '@material-ui/core/Grid';
+import Avatar from '@material-ui/core/Avatar';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+
+interface Props {
+  notifications: Notification[];
+  type: 'page' | 'dropdown';
+}
+export function NotificationsList({ notifications, type }: Props): JSX.Element {
+  const classes = useStyles();
+  // TODO: Mark notification as read when clicked
+  // const [invisible, setInvisible] = React.useState(false);
+
+  // const handleBadgeVisibility = () => {
+  //   setInvisible(true);
+  // };
+  return (
+    <List component="nav" className={classes.root} aria-label="notifications">
+      {notifications.map(
+        (notification: {
+          _id: string;
+          type: string;
+          senderId: {
+            _id: string;
+            username: string;
+          };
+          receiverId: string;
+          contestId: {
+            _id: string;
+            title: string;
+          };
+          submissionId?: string;
+          timeSent: Date;
+          readStatus: boolean;
+          photo: string;
+        }) => (
+          <>
+            <ListItem
+              button
+              key={(notification.type, notification.photo, notification.senderId.username, notification.type)}
+            >
+              <Grid container={true} alignItems="center" justify="center" spacing={2} wrap="nowrap">
+                <Grid item xs={2} container direction="column" justify="flex-end" alignItems="flex-end">
+                  <Avatar alt="Notification Thumbnail" src={notification.photo} />
+                </Grid>
+                <Grid item xs={9}>
+                  <Typography variant="body2" noWrap={type === 'dropdown' ? true : false}>
+                    {notification.type === 'submission' ? (
+                      <span>
```

Comment:
Maybe use MUI components here to stay more consistent with design? :)

Ref: https://material-ui.com/system/typography/